$NetBSD$

Don't build python bindings and tools.
Don't look for the dependencies needed by
the things not built.

--- CMakeLists.txt.orig	2023-07-02 02:16:37.000000000 +0000
+++ CMakeLists.txt
@@ -119,10 +119,6 @@ set(OpenGL_GL_PREFERENCE GLVND)
 
 ## Search for useful libraries
 find_package(Threads REQUIRED)
-find_package(GLUT REQUIRED)
-find_package(OpenGL REQUIRED)
-find_package(PythonInterp REQUIRED)
-find_package(PythonLibs REQUIRED)
 find_package(ZLIB)
 if (ZLIB_FOUND)
     add_definitions(-DPARTIO_USE_ZLIB)
@@ -135,9 +131,9 @@ set(PARTIO_LIBRARIES partio ${ZLIB_LIBRA
 
 ## Traverse subdirectories
 add_subdirectory(src/lib)
-add_subdirectory(src/tools)
-add_subdirectory(src/py)
-add_subdirectory(src/doc)
+#add_subdirectory(src/tools)
+#add_subdirectory(src/py)
+#add_subdirectory(src/doc)
 
 if (${PARTIO_GTEST_ENABLED})
     set(GTEST_LOCATION "/usr" CACHE STRING "gtest installation prefix")
